src/js/containers/SecurityPage/SecurityPage.js

```diff
@@ -1,55 +0,0 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { FormattedMessage, injectIntl } from 'react-intl';
-import _ from 'lodash';
-
-import { Heading } from 'cf-component-heading';
-
-import BrowserIntegrityCheckCard from '../../containers/BrowserIntegrityCheckCard/BrowserIntegrityCheckCard';
-import ChallengePassageCard from '../../containers/ChallengePassageCard/ChallengePassageCard';
-import FeatureManager from '../../components/FeatureManager/FeatureManager';
-import SecurityLevelCard from '../../containers/SecurityLevelCard/SecurityLevelCard';
-import SSLCard from '../../containers/SSLCard/SSLCard';
-
-class SecurityPage extends Component {
-
-    render() {
-        let { activeZoneId, zoneSettings } = this.props;
-        let isSettingsEmpty = _.isEmpty(zoneSettings.entities[activeZoneId]);
-
-        return (
-            <div>
-                {isSettingsEmpty && (<FormattedMessage id="errors.noActiveZoneSelected"/>)}
-                {!isSettingsEmpty && (
-                    <div>
-                        <Heading size={1}><FormattedMessage id="container.securityPage.title"/></Heading>
-                        <FeatureManager isEnabled={this.props.config.featureManagerIsSSLEnabled}>
```

Comment:
Eventually we can remove it and convert it to a selector.  Don't worry about it for now.